### PR TITLE
26-Dedicated Registration Page

### PR DESCRIPTION
**Pull request by RegisterRonny**
Added a dedicated Registration Page to the main menu that displays the
Registration widget.

### CODE REVIEW

File: templates/views/registration.hbs [hunk 1]
```diff
@@ -0,0 +1,12 @@
+<div class="container"><h1>{{register.name}}</h1></div>
```
Comment (gravling) [inline templates/views/registration.hbs]: Remove this line before merge

**Comment by gravling** [comment]
@rogsa are you sure about the 8 column span in `md`?

**Comment by rogsa** [comment]
No, that's to narrow. Passing it on to @RegisterRonny, who made the registration page. It should be 12-column. And with the same padding/margin and background-color as article.base-page. Looks best under current layout/coloring, i think.

**Comment by rogsa** [comment]
The registration page is in a single container with "col-xs-12", which may not be the best option from a responsive perspective but it will work fine for now. I have some plans in making the whole application more responsive so that it in "narrow" mode it will disregard positioning and spanning from form configuration and layout all fields on single lines with 100% width. In the "narrowest" mode it will also place event panel first and form under. Reducing paddings and margins also. Problem is that layout is server driven. Hmmm. Could be interesting to try, though :-)

**Pull request merged**